app/config/config_test.yml

```diff
@@ -111,3 +111,10 @@ old_sound_rabbit_mq:
 
 lexik_paybox:
     public_key: "%env(SSL_PUBLIC_KEY)%"
+
+doctrine:
+    dbal:
+        connections:
```

Comment:
you may also consider using at least an arrayCache pour doctrine metadata ;)

Reply:
We will do that with [DoctrineTestBundle](https://github.com/dmaicher/doctrine-test-bundle) in another PR